Antidote/CallsManager.m

```diff
@@ -222,6 +222,36 @@ - (void)updateCurrentCallInterface:(OCTCall *)call
         else {
             activeVC.callDuration = call.callDuration;
         }
+
+
+        if (call.videoIsEnabled && ! activeVC.previewLayer) {
+            // Have to figure out a better way to do this so we don't have to give
+            // a dummy layer. Otherwise thread will keep calling previewLayer.
+            // Even with the below, getting or removing the the preview layer
+            // is locking up the main queue.
+            //            CALayer *dummyLayer = [CALayer new];
+            //            activeVC.previewLayer = dummyLayer;
+            //
+            //            __weak ActiveCallViewController *weakVC = activeVC;
+            //            [self.manager getVideoCallPreview:^(CALayer *layer) {
+            //                ActiveCallViewController *strongVC = weakVC;
+            //                strongVC.previewLayer = layer;
+            //            }];
```

Comment:
Yea maybe I should do the same as well for `videoView`? That way we can go back to using methods like you suggested on `ActiveCallViewController` instead of properties.